### PR TITLE
Samlinf

### PR DESCRIPTION
**Pull request by u9510136**
service bus works between Room and Complex API

### CODE REVIEW

File: complex/.docker/dockerup.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ services:
       - complex_localhost
     environment:
       ConnectionStrings__ComplexDb: 'Host=data;Database=complex;Username=postgres;Password=Pass@word'
+      ConnectionStrings__ServiceBus: 'Endpoint=sb://gabservice.servicebus.windows.net/;SharedAccessKeyName=project2policy;SharedAccessKey=9gdvy0ZlZ8kBRYqwe27uP4cmx+G20udOj+/wzAYvQvs='
```
Comment (mariellenolasco) [inline complex/.docker/dockerup.yaml]: this shouldn't be here

File: complex/src/Revature.Complex.Api/appsettings.json [hunk 2]
```diff
@@ -6,7 +6,6 @@
   "Queues": {
     "DeletedRooms": "DeletedRooms",
     "CRUDRoom": "CRUDRoom",
-    "AddAddress": "AddAddress"
   },
```
Comment (mariellenolasco) [inline complex/src/Revature.Complex.Api/appsettings.json]: Did nick approve this change to appsettings.json?

Comment (mariellenolasco) [reply complex/src/Revature.Complex.Api/appsettings.json]: I think its best if you leave the appsettings.json file as is

File: complex/.docker/dockerup.yaml (hunk 1 above)
Comment (u9510136) [reply complex/.docker/dockerup.yaml]: deleted

File: complex/src/Revature.Complex.Api/appsettings.json (hunk 2 above)
Comment (u9510136) [reply complex/src/Revature.Complex.Api/appsettings.json]: added back

File: complex/.docker/dockerup.yaml [hunk 3]
```diff
@@ -15,7 +15,8 @@ services:
     networks:
       - complex_localhost
     environment:
-      ConnectionStrings__ComplexDb: 'Host=data;Database=complex;Username=postgres;Password=Pass@word'
+      ConnectionStrings__ComplexDb: ''
+      ConnectionStrings__ServiceBus: ''
   data:
```
Comment (mariellenolasco) [inline complex/.docker/dockerup.yaml]: Don't change the yaml file from master at all

Comment (u9510136) [reply complex/.docker/dockerup.yaml]: fixed

File: complex/src/Revature.Complex.Api/appsettings.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
-    "ComplexDb": "(provide connection string)",
-    "ServiceBus": "(provide connection string)"
+    "SerciceBus": "(provide connection string)",
+    "ComplexDb": "(provide connection string)"
```
Comment (mariellenolasco) [inline complex/src/Revature.Complex.Api/appsettings.json]: ServiceBus not SerciceBus

Comment (u9510136) [reply complex/src/Revature.Complex.Api/appsettings.json]: fixed

File: complex/.docker/dockerup.yaml (hunk 3 above)
Comment (mariellenolasco) [reply complex/.docker/dockerup.yaml]: just look at the docker yaml in master. It shouldn't register as a change

**Comment by escalonn** [comment]
merge master

**Pull request merged**